src/web/AlertModalContent.tsx

```diff
@@ -131,9 +132,9 @@ export class AlertModalContent extends RX.Component<AppModalContentProps, AppMod
                         onPress={ e => this._onPressButton(btnSpec) }
                         onHoverStart={ () => this.setState({ hoverIndex: i }) }
                         onHoverEnd={ () => this.setState({ hoverIndex: -1 }) }
-                        style={ buttonStyle }
+                        style={ _.compact(buttonStyle) }
```

Comment:
Why are you compacting these styles? It's perfectly legit for style arrays to contain undefined values.